tests/test_interpreter.py

```diff
@@ -1,16 +1,18 @@
+import sys
+import os
 from random import randint
 import interpreter
 import time
 
-interpreter.auto_run = True
-interpreter.model = "gpt-3.5-turbo"
-interpreter.temperature = 0
-
 
 # this function will run before each test
 # we're clearing out the messages Array so we can start fresh and reduce token usage
 def setup_function():
     interpreter.reset()
+    interpreter.temperature = 0
+    interpreter.auto_run = True
+    interpreter.model = "gpt-3.5-turbo"
+    interpreter.debug_mode = False
```

Comment:
Now that we're testing configuration overrides by loading a custom config file, we need to make sure that our setup function resets to the expected parameters for our tests.